scipy/stats/tests/test_stats.py

```diff
@@ -1771,6 +1771,30 @@ def test_zscore_ddof(self):
         assert_array_almost_equal(z[0], z0_expected)
         assert_array_almost_equal(z[1], z1_expected)
 
+    def test_zscore_nan_propagate(self):
+        # Test nan_policy set to 'propogate'
+        x = np.array([1, 2, np.nan, 4, 5])
+
```

Comment:
Silly nitpick: might as well leave out the blank lines here. Per Pep 8:

"Use blank lines in functions, sparingly, to indicate logical sections"

and I think this test is all one "logical section".

Reply:
Thank you.